taxi/views.py

```diff
@@ -40,15 +42,60 @@
     queryset = Car.objects.all().select_related("manufacturer")
 
 
-class CarDetailView(LoginRequiredMixin, generic.DetailView):
+class CarDetailView(LoginRequiredMixin, DetailView):
+    model = Car
+    template_name = "taxi/car_detail.html"
+    context_object_name = "car"
+
+
+class CarCreateView(CreateView):
+    model = Car
+    form_class = CarForm
+    template_name = "taxi/car_form.html"
+    success_url = reverse_lazy("taxi:car-list")
+
+
+class CarUpdateView(UpdateView):
+    model = Car
+    form_class = CarForm
+    template_name = "taxi/car_form.html"
+    success_url = reverse_lazy("taxi:car-list")
+
+
+class CarDeleteView(DeleteView):
     model = Car
+    template_name = "taxi/car_confirm_delete.html"
+    success_url = reverse_lazy("taxi:car-list")
+
+
+class ManufacturerCreateView(CreateView):
+    model = Manufacturer
+    form_class = ManufacturerForm
+    template_name = "taxi/manufacturer_form.html"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+
+
+class ManufacturerUpdateView(UpdateView):
+    model = Manufacturer
+    form_class = ManufacturerForm
+    template_name = "taxi/manufacturer_form.html"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+
+
+class ManufacturerDeleteView(DeleteView):
+    model = Manufacturer
+    template_name = "taxi/manufacturer_confirm_delete.html"
+    success_url = reverse_lazy("taxi:manufacturer-list")
 
 
 class DriverListView(LoginRequiredMixin, generic.ListView):
     model = Driver
+    template_name = "taxi/driver_list.html"
     paginate_by = 5
+    context_object_name = "driver_list"
```

Comment:
You don't need to override the context_object_name if it is the same as the default one. In this case, the default name is 'driver_list', which is the same as the one you've set. So you can safely remove this line.